src/creds/creds.c

```diff
@@ -636,11 +636,13 @@ static int verb_has_tpm2(int argc, char **argv, void *userdata) {
                 printf("%sfirmware\n"
                        "%sdriver\n"
                        "%ssystem\n"
-                       "%ssubsystem\n",
+                       "%ssubsystem\n"
+                       "%slibraries\n",
```

Comment:
The change to creds.c should be in the first commit?

Reply:
Ugh I keep forgetting to rebase instead of amend